### PR TITLE
Update Vietnamese translation

### PR DESCRIPTION
**Pull request by datlechin**
**Details of Changes**
- Update translations for Vietnamese (vi).

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@datlechin before we merge this, can you add your agreement to #7625?

File: src/Files/MultilingualResources/Files.vi.xlf [hunk 1]
```diff
@@ -2928,83 +2928,83 @@
         </trans-unit>
         <trans-unit id="DateFormat" translate="yes" xml:space="preserve">
           <source>Date format</source>
-          <target state="new">Date format</target>
+          <target state="new">Định dạng ngày tháng</target>
```
Comment (yaichenbaum) [inline src/Files/MultilingualResources/Files.vi.xlf:2931]: Can you update the state to `translated`? https://files.community/docs/contributing/translating-files

Comment (datlechin) [reply src/Files/MultilingualResources/Files.vi.xlf]: Thanks, I changed them to `translated`

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Comment by yaichenbaum** [comment]
@datlechin before we merge this, can you add your agreement to #7625?

**Pull request merged**